gengokit/handler/handler.go

```diff
@@ -241,11 +237,10 @@ func isValidFunc(f *ast.FuncDecl, m methodMap, pkgName string) bool {
 	return true
 }
 
-// mRecvTypeString returns accepts and *ast.FuncDecl.Recv recv, and returns the
+// mRecvTypeString accepts an *ast.FuncDecl.Recv recv, and returns the
```

Comment:
Curious about the naming convention here. This looks like a throwback to C++ when some programmers used this convention to indicate that a variable was a class member. I'm not sure what it means here. Is there a good reason for it, or should it be renamed?